packages/Avalonia/AvaloniaRules.Project.xml

```diff
@@ -0,0 +1,81 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Rule Name="Avalonia" DisplayName="Avalonia" Order="70" PageTemplate="generic" xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.Categories>
+    <Category Name="Compiling" DisplayName="Compiling" />
+    <Category Name="Debug" DisplayName="Debugging" />
+    <Category Name="NameGenerator" DisplayName="Nameming Generator" />
```

Comment:
I would suggest "Compilation"/"Compile", "Debug" and "Name Generator" for the category group names. I'm not sure verbs are best here.

Note that "Nameming" is a spelling error regardless and should be "Naming"